app/src/main/java/de/danoeh/antennapod/fragment/preferences/NetworkPreferencesFragment.java

```diff
@@ -174,4 +152,145 @@ private void showUpdateIntervalTimePreferencesDialog() {
         }
         return entries;
     }
+
+    private void FeedRefreshIntervalDialog()
```

Comment:
Could you please move this to a new class? That could reduce the size of this already pretty long preference screen class.